apps/src/lib/kits/maker/commands.js

```diff
@@ -106,7 +107,8 @@ export function boardConnected() {
  */
 export function onBoardEvent(opts) {
   let {component, event, callback} = opts;
-  if (BOARD_EVENT_ALIASES[event]) {
+  // Look up aliases for CircuitPlaygroundBoard
+  if (BOARD_EVENT_ALIASES[event] && !(board instanceof MicroBitBoard)) {
```

Comment:
why don't we want to do this lookup for micro:bit boards?

Reply:
We surface the 'shake' event to the student, however, the CircuitPlayground doesn't recognize that event, so we alias it to singleTap event that the board recognizes. MicroBit handles the shake event (and not the tap events) so we don't need to alias it. The FakeBoard is just a CP FakeBoard, so we need to alias for the CP and FakeCP, but not the MicroBit.

Reply:
ohhh, makes sense. thank you!